CODEOWNERS

```diff
@@ -209,6 +209,7 @@
 /bundles/org.openhab.binding.modbus.sbc/ @fwolter
 /bundles/org.openhab.binding.modbus.stiebeleltron/ @pail23
 /bundles/org.openhab.binding.modbus.studer/ @giovannimirulla
+/bundles/org.openhab.binding.modbus.sungrow/ @soenkekueper@gmx.de
```

Comment:
This is expected to be GitHub username, not e-mail address.

Reply:
fixed